### DISCUSSION TITLE
Manifest 3 and Group Speed Dial

### DISCUSSION THREAD

**Discussion by IAmHugh001** [fastaddons/GroupSpeedDial#336] [General]
LOVE Group Speed Dial and hope to keep using it. I just got a email from the Simkl devs indicating we are less than 2 weeks away from Manifest 3 and according to them Manifest 2 extensions will not work with it. I'd like to know if this is going to affect Group Speed Dial. Thanks

https://forum.vivaldi.net/topic/93057/how-will-vivaldi-deal-with-google-s-manifest-v3/101?_=1720350986730

Again GREAT extension.

**Comment by Juraj-Masiar** [comment]
Hello,
No worries, it's already using Manifest V3 in Chrome store :)
I've migrated it already 2 years ago since the first deadline was already last year, but then they moved it.
Some extensions are easier to migrate than others...
Have a nice upcoming week :)

**Comment by IAmHugh001** [comment]
Thanks good to know.


Sent using {0}








---- On Sun, 07 Jul 2024 13:26:07 -0700 Juraj Mäsiar ***@***.***> wrote ---





Hello,
 No worries, it's already using Manifest V3 in Chrome store :)
 I've migrated it already 2 years ago since the first deadline was already last year, but then they moved it.
 Some extensions are easier to migrate than others...
 Have a nice upcoming week :)
